pkg/config/controller_config.go

```diff
@@ -11,13 +12,16 @@ import (
 const (
 	LatticeGatewayControllerName = "application-networking.k8s.aws/gateway-api-controller"
 	defaultLogLevel              = "Info"
+	NoDefaultServiceNetwork      = ""
+	NO_DEFAULT_SERVICE_NETWORK   = "NO_DEFAULT_SERVICE_NETWORK"
 )
 
 // TODO endpoint, region
 var VpcID = "vpc-xxxx"
 var AccountID = "yyyyyy"
 var Region = "us-west-2"
 var logLevel = defaultLogLevel
+var DefaultServiceNetwork = NoDefaultServiceNetwork
 
```

Comment:
Nit:

Could we remove `NoDefaultServiceNetwork`  and just use: 

```
var DefaultServiceNetwork = ""
if DefaultServiceNetwork == ""
```
That have better readability for me